### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by EdSacasa**
Added better description on README file

### CODE REVIEW

File: index.html
Comment (wathigo) [inline index.html:12]: **[REQUIRED]** Put the navigation bar inside the `header` element or remove it.

File: index.html
Comment (wathigo) [inline index.html]: **[OPTIONAL]** Use `nav` element instead of `section` for the navigation bar. Read more about [semantic html](https://www.w3schools.com/html/html5_semantic_elements.asp)

**Review by wathigo — CHANGES_REQUESTED** [review]
## Hi @EdSacasa  and @kazumaki  :wave: 
**Great work so far,** 
The form looks great.
Before we get this approved, we need to fix a couple of things:
1. Your CSS code contains errors. Upload your code to [w3 validator](https://validator.w3.org/#validate_by_upload) and fix all the errors.
2. All files that are not important like `.DS_Store` and `.Rhistory` should be included in `.gitignore` file and not commited. 
3. Make sure you understand [github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). In github flow, we don't have `develop` branch. We only have a `feature-branch`.
4. Give this Pull Request a title and a description of the accomplished tasks
5. Implement the changes below :point_down: 

**After implementing the changes, submit another request for review** 
_Ensure you have balanced number of commits between the two of you. 
Good luck :)_

File: index.html
Comment (Torres-ssf) [inline index.html]: [REQUIRED]
Instead of using only one image for the 3 logos(mint, quickbooks, turbotax), use the 3 logos(you can find them in the original page) with text for the names of the products.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Hi there team :hand: 
Nice job implementing these changes, **but we still have something to change**.

- [x] Instead of using only one image for the 3 logos(mint, quickbooks, turbotax), use the 3 logos(you can find them in the original page) with text for the names of the products.

Kindly make this change and submit for code review again.

### **IMPORTANT**
**[FOR NEXT PROJECTS]**

- For all HTML & CSS projects, use only **one** branch and do not merge it to master until it's been approved.

- As it said in the pathwrite, you should use Github Flow For all HTML and CSS projects. So give a feature name to your branch (e.g. embedding-images-and-video, html-forms, responsive-design, etc)  instead of develop/dev. You can read more about this [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/).

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job so far :thumbsup: @EdSacasa and @kazumaki 
Look at the following request before the project gets approved. 
- [x] Fix all the conflicts in  ```.stickler.yml```, ```index.html``` and style.css 

Please when complete submit for another review.

**Review by whiz25 — APPROVED** [review]
## Awesome job :clap: 
Your project meets all checklists.
Please go ahead and submit a project completions form.
I wish you success in your next projects.

**Pull request merged**
